Patch-Source: https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/commit/?id=bd318469612b9c686144e005eddcebd92c08a845
added right after 20221012
--
From bd318469612b9c686144e005eddcebd92c08a845 Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Sun, 16 Oct 2022 13:50:44 +0100
Subject: brcm: add symlink for Pi Zero 2 W NVRAM file

The Raspberry Pi Zero 2 W comes with two possible WiFi modules.
One of them is the same module as shipped in the original
Raspberry Pi 3B and Zero W so lets link them so the devices
with that module will work out of the box.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
Signed-off-by: Josh Boyer <jwboyer@kernel.org>
---
 WHENCE | 1 +
 1 file changed, 1 insertion(+)

diff --git a/WHENCE b/WHENCE
index 0ffb517..205197a 100644
--- a/WHENCE
+++ b/WHENCE
@@ -2857,6 +2857,7 @@ File: "brcm/brcmfmac43430-sdio.Hampoo-D2D3_Vi8A1.txt"
 File: "brcm/brcmfmac43430-sdio.MUR1DX.txt"
 File: "brcm/brcmfmac43430-sdio.raspberrypi,3-model-b.txt"
 Link: brcm/brcmfmac43430-sdio.raspberrypi,model-zero-w.txt -> brcmfmac43430-sdio.raspberrypi,3-model-b.txt
+Link: brcm/brcmfmac43430-sdio.raspberrypi,model-zero-2-w.txt -> brcmfmac43430-sdio.raspberrypi,3-model-b.txt
 File: "brcm/brcmfmac43455-sdio.acepc-t8.txt"
 File: "brcm/brcmfmac43455-sdio.raspberrypi,3-model-b-plus.txt"
 Link: brcm/brcmfmac43455-sdio.raspberrypi,3-model-a-plus.txt -> brcmfmac43455-sdio.raspberrypi,3-model-b-plus.txt
-- 
cgit 

